mappings/net/minecraft/block/Material.mapping

```diff
@@ -3,54 +3,76 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15912 breakByHand Z
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
+		COMMENT Material for blocks that are stone or made from it, and generally prefer to be broken by a pickaxe.
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
+		COMMENT A material or structure formed from a loosely compacted mass of fragments or particles.
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
+		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
+		COMMENT Not in use, but has foliage color.
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 SUPPORTED Lnet/minecraft/class_3614;
```

Comment:
Since this is used for skulls, supported doesn't work. I'd go with PART or COMPONENT.

Reply:
@Prospector https://github.com/FabricMC/yarn/pull/1190#discussion_r415417695

Reply:
That was before I realized skulls were part of it, which aren't "supported." Also, support isn't a property of materials so tbh thinking about it more it really doesn't make sense.